r-package/grf/bindings/RcppUtilities.cpp

```diff
@@ -50,13 +55,14 @@ Forest RcppUtilities::deserialize_forest(const Rcpp::List& forest_object) {
   return Forest(trees, num_variables, ci_group_size);
 }
 
-Rcpp::List RcppUtilities::serialize_forest(Forest& forest) {
+Rcpp::List RcppUtilities::serialize_forest(SEXP forest_xptr) {
+  Rcpp::XPtr<Forest> forest(forest_xptr);
   Rcpp::List result;
 
-  result.push_back(forest.get_ci_group_size(), "_ci_group_size");
-  result.push_back(forest.get_num_variables(), "_num_variables");
+  result.push_back((*forest).get_ci_group_size(), "_ci_group_size");
```

Comment:
Generally instead of doing `(*object).method()`, you can write `object->method()`.